ui/package.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://mozilla-services.github.io/buildhub",
   "scripts": {
     "deploy":
-      "yarn run build && rimraf tmp && mkdir tmp && cp -R build/* tmp/ && cp -R build/* tmp/stage/ && ls tmp/stage/static/js/main* tmp/stage/index.html | xargs sed -i 's|/buildhub/|/buildhub/stage/|g' && ls tmp/stage/static/js/main* | xargs sed -i 's/buildhub.prod.mozaws.net/buildhub.stage.mozaws.net/' && gh-pages -d tmp --add && rimraf tmp",
+      "yarn run build && rimraf tmp && mkdir tmp && cp -R build/* tmp/ && gh-pages -d tmp --add && rimraf tmp",
```

Comment:
I took out all the stage-related things. There's a ton of stuff here and it's really hard to debug it in this form.

Given that we've deprecated Buildhub, I'm simplifying it so as to get the deprecation notice out.